tests/testsuite/script.rs

```diff
@@ -1886,3 +1886,54 @@ CARGO_MANIFEST_PATH: [ROOT]/foo/script.rs
 "#]])
         .run();
 }
+
+#[cargo_test(nightly, reason = "edition2024 hasn't hit stable yet")]
+fn ignore_surrounding_workspace() {
```

Comment:
We ask that all commits be atomic which includes tests passing.

If you could update this commit so that the test passes, that'd be appreciated!  We do like having the test commit split out like this.  In doing so, the diff for the fix commit shows how the behavior was fixed (e.g. #15497)

Reply:
Sure, test now passes in both commits.